definitions/divorce/json/CaseField/CaseField.json

```diff
@@ -1745,7 +1745,7 @@
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "DIVORCE",
     "ID": "LabelSolPaymentPara-1",
-    "Label": "Amount to pay: **£550**",
+    "Label": "Amount to pay: **£${solApplicationFeeInPounds}**",
```

Comment:
Looks good, and I guess you set this value using a callback?

Reply:
Thanks. That's right.